ecommerce-app/src/components/auth/LoginPageContent.tsx

```
@@ -4,53 +4,48 @@ import { useRouter } from 'next/router';
+  useEffect(() => {
+    setIsClient(true);
+  }, []);
+
+  // 클라이언트에서만 토큰 체크 및 리다이렉트
+  useEffect(() => {
+    if (!isClient) return;
+
+    // 로그인 유저면 리다이렉트
+    if (false) {
+      // 실제 운용에서는 API로 토큰 검증이 필요하지만 현재는 토큰 존재만 확인
+      // TODO: 추후 토큰 디코딩으로 role 확인 구현
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)